solutions/secure/outputs.tf

```diff
@@ -0,0 +1,37 @@
+##############################################################################
+# Outputs
+##############################################################################
+output "resource_group_id" {
+  description = "Resource Group ID"
+  value       = module.resource_group.resource_group_id
+}
+
+output "buckets" {
+  description = "List of buckets created"
+  value       = module.cos_fscloud_da.buckets
```

Comment:
Outputs value will be changed after changing the module name. This comment is valid for other outputs vars too.